x-pack/docs/en/security/authentication/oidc-guide.asciidoc

```diff
@@ -552,8 +551,8 @@ for this Kibana instance.
 {kib} has in-built protection against _Cross Site Request Forgery_ attacks, which
 is designed to prevent the {kib} server from processing requests that
 originated from outside the {kib} application.
-In order to support OpenID Connect messages that originate from your
-OP or a third party (see <<third-party-login>>, we need to explicitly _whitelist_ the
+In case you use the authorization code flow, {kib} needs to support unsolicited messages that originate from your
+OP or a third party (see <<third-party-login>>). In order to do so, we need to explicitly _whitelist_ the
```

Comment:
Who is "we" in this case, Kibana? Or should it be changed to "you must explicitly..."?

> In order to do so, we need to ...

Reply:
>  Or should it be changed to "you must explicitly..."?
Yep, thanks!